### ISSUE TITLE
[strands_movebase] Obstacles lingering at edge of laser FOV

### ISSUE THREAD

**Issue by nilsbore** [strands-project/strands_movebase#25]
As discussed in https://github.com/strands-project/strands_navigation/issues/164#issuecomment-64439056, there seems to be obstacles lingering at the laser's FOV. I think the way to solve this is to add a node that takes the sensor input and cuts off the FOV. Then `move_base` should use only this slightly narrower FOV to add obstacles and the original sensor input to clear them. This is similar to how it's done for the chest camera.

**Comment by nilsbore** [comment]
I have created a fix for this according to the outline above in my `laser_devel` branch: https://github.com/nilsbore/strands_movebase/tree/laser_devel . The creation of this the new laser scan makes some approximations, but it seems good enough for now. I will try this out whenever Rosie crashes. In the meantime, I have tried out the node creating the new laser scan on Rosie and it works. So anyone that feels that this could scratch their itch can feel to try it out. @BFALacerda ?
